web/beacon-app/src/main.tsx

```diff
@@ -8,8 +8,10 @@ import { RouterProvider } from 'react-router-dom';
 import { queryClient, QueryClientProvider } from '@/application/config/react-query';
 import router from '@/application/routes/root';
 
-import initSentry from './config/sentry';
+import { appConfig } from './application/config';
+import initSentry from './application/config/sentry';
 
+console.log('initializing beacon ui', appConfig.nodeENV, appConfig.version, appConfig.revision);
```

Comment:
Added some debugging information to the console so it would be easier for us to understand what's happening in deployments.